Model/Dashboard.php

```diff
@@ -54,6 +54,11 @@
     protected $edited;
 
     /**
+     * @var bool
+     */
+    protected $default;
```

Comment:
Because every property in model classes are `proptected`, so someone could use it in their entity.